src/sentry/static/sentry/app/views/performance/landing.tsx

```diff
@@ -348,40 +327,15 @@ class PerformanceLanding extends React.Component<Props, State> {
                   setError={this.setError}
                 />
               ) : (
-                <div>
-                  <StyledSearchBar
-                    organization={organization}
-                    projectIds={eventView.project}
-                    query={filterString}
-                    fields={generateAggregateFields(
-                      organization,
-                      [...eventView.fields, {field: 'tps()'}],
-                      ['epm()', 'eps()']
-                    )}
-                    onSearch={this.handleSearch}
-                  />
-                  <Feature features={['performance-vitals-overview']}>
-                    <VitalsCards
-                      eventView={eventView}
-                      organization={organization}
-                      location={location}
-                    />
-                  </Feature>
-                  <Charts
-                    eventView={eventView}
-                    organization={organization}
-                    location={location}
-                    router={router}
-                  />
-                  <Table
-                    eventView={eventView}
-                    projects={projects}
-                    organization={organization}
-                    location={location}
-                    setError={this.setError}
-                    summaryConditions={summaryConditions}
-                  />
-                </div>
+                <LandingContent
+                  eventView={eventView}
+                  projects={projects}
+                  organization={organization}
+                  location={location}
+                  router={router}
```

Comment:
You could use `withRouter()` to get the router and location provided as props instead of having to forward them down.